infra/modules/database/role_manager/role_manager.py

```diff
@@ -114,16 +119,27 @@ def connect_using_iam(user: str) -> Connection:
     logger.info("Connecting to database: user=%s host=%s port=%s database=%s", user, host, port, database)
     return Connection(user=user, host=host, port=port, database=database, password=token, ssl_context=True)
 
+import requests
```

Comment:
I'm not seeing this being used anywhere, is it safe to delete?

Reply:
It was used for testing. Removing